features/steps/fakeapi.go

```diff
@@ -42,3 +44,14 @@ func (f *FakeAPI) setJSONResponseForGetPageData(url string, pageType string, sta
 	}
 	f.fakeHTTP.NewHandler().Get(path).Reply(statusCode).BodyString(bodyStr)
 }
+
+func (f *FakeAPI) setJSONResponseForGetBreadcrumb(url string, status int) {
+	//specialCharUrl := strings.Replace(url, "/", "%2F", -1)
+	path := "/parents?uri=" + url
+	bodyStr := `[
+   		{"uri": "/", "description": {"title": "Home"}, "type": "home_page"}, 
+   		{"uri": "/economy", "description": {"title": "Economy"}, "type": "taxonomy_landing_page"},
+   		{"uri": "/economy/grossdomesticproductgdp", "description": {"title":"Gross Domestic Product (GDP)"}, "type": "product_page"}
+	]`
```

Comment:
This could be more configurable, but appreciate this is a first take.

Reply:
It could be.